types/priv_validator/socket.go

```diff
@@ -71,43 +104,74 @@ func (pvsc *PrivValidatorSocketClient) OnStop() {
 	}
 }
 
-func (pvsc *PrivValidatorSocketClient) Address() data.Bytes {
-	pubKey := pvsc.PubKey()
-	return pubKey.Address()
+// Address is an alias for PubKey().Address().
+func (pvsc *PrivValidatorSocketClient) Address() cmn.HexBytes {
+	return pvsc.PubKey().Address()
 }
 
+// PubKey implements PrivValidator.
 func (pvsc *PrivValidatorSocketClient) PubKey() crypto.PubKey {
```

Comment:
Should not PubKey return error too by analogy with `SignProposal`?

Reply:
Thought so too and addressed it in 2a292ef, but would like to have a sanity check from @ebuchman on this. :>